scipy-1.0/paper.tex

```diff
@@ -735,52 +735,52 @@ \subsection*{Submodule organization}
 
 \subsection*{Language choices}
 
-Python, Cython, Fortran, C and C++ are the programming languages used to
-implement scientific algorithms in the SciPy library. An analysis of our code
-base using the \texttt{linguist} library\cite{linguistref} provides a 
-detailed breakdown as \% composition by programming language in 
-SciPy (Figure~\ref{fig:linguist}).
+Figure~\ref{fig:linguist} provides a breakdown of the programming languages used
+in SciPy. While it is not surprising that Python is predominant, the usage distribution 
+of the other languages warrants discussion. 
 
 \begin{figure}[H]
-    \centering
-
-    \begin{tikzpicture}[font=\sffamily]
-    \begin{axis}[height=6cm, width=10cm,
-    ybar, ymax=60, ymin=0,bar width=1cm,
-    ylabel={\% of Scipy code base},
-    ytick={0,20,...,60},
-    axis lines*=left,
-    axis y line shift=0pt,
-    enlarge x limits=0.15,
-    symbolic x coords={py,f,c,cy,cpp},
-    xticklabels={Python,FORTRAN,C,Cython,C++},
-    ticklabel style={align=center,font=\sffamily\small,
-    /pgf/number format/assume math mode=true},
-    xtick=data,
-    nodes near coords,
-    nodes near coords style={/pgf/number format/assume math mode=true}
-    ]
-    \addplot coordinates {(py,49.5) (f,25.6) (c,19.5) (cy, 3.0) (cpp,2.3)};
-    \end{axis}
-    \end{tikzpicture}
-    \caption{The breakdown of programming languages used in the
-             SciPy library determined using the linguist library.
-    	 Small ($<0.5 \%$) amounts of TeX, Matlab, Shell,
-    	 and Makefile are excluded for clarity and mostly
-    	 provide supporting roles in tests, building, and
-    	 documentation.}
-    \label{fig:linguist}
+\centering
+\includegraphics[width=0.5\textwidth]{static/scipy_languages}
+\caption{Composition of SciPy: lines of code in each programming language as determined by the \texttt{linguist} package.
```

Comment:
It is not lines of code, it is bytes; the linguist repo points to this: https://developer.github.com/v3/repos/#list-languages

This may also be a source of confusion in the related PR about coverage vs. the data here.

Reply:
Oops, I got the two mixed up. I'll fix that. The idea is to address the apparent discrepancy and try to explain it.